core/src/main/java/org/elasticsearch/index/query/QueryStringQueryBuilder.java

```diff
@@ -266,6 +271,7 @@ protected void doWriteTo(StreamOutput out) throws IOException {
         this.fuzziness.writeTo(out);
         out.writeVInt(this.fuzzyPrefixLength);
         out.writeVInt(this.fuzzyMaxExpansions);
+        out.writeBoolean(this.fuzzyTranspositions);
```

Comment:
Hi Nik!

Would this work? Unsure what the field should default - the DEFAULT value is taken from `FuzzyQuery.defaultTranspositions`.
```   if (in.getVersion().onOrAfter(Version.V_5_1_1_UNRELEASED)) {
            fuzzyTranspositions = in.readBoolean();
        } else {
            fuzzyTranspositions = DEFAULT_FUZZY_TRANSPOSITIONS;
        }```